.github/workflows/cypress.yml

```diff
@@ -13,6 +13,9 @@ jobs:
         with:
           bundler-cache: true
 
+      - name: Run migrations
+        run: bin/rails db:migrate RAILS_ENV=development
```

Comment:
```suggestion
        run: bin/rails db:migrate RAILS_ENV=test
```
This should be `test` or this is the bug we talked about?

Reply:
This should be development